client/cypress/e2e/settings.cy.ts

```diff
@@ -10,8 +10,12 @@ import {
 } from 'src/constants/localStorageKeys';
 import { ROOT_ROUTES } from 'src/routes/RootRoutes/routes';
 import getValueCryptoToDisplay from 'src/utils/getValueCryptoToDisplay';
+import { OCTANT_BUILD_LINK } from '../../src/constants/urls.ts';
+import { OCTANT_DOCS } from '../../src/constants/urls.ts';
+import { DISCORD_LINK } from '../../src/constants/urls.ts';
+import { TERMS_OF_USE } from '../../src/constants/urls.ts';
```

Comment:
Please import 4 elements from one file using absolute path instead of 4 imports using relative paths.

Reply:
ok